board/opentrons/ot2/rootfs-overlay/etc/avahi/services/opentrons.service

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" standalone='no'?><!--*-nxml-*-->
+<!DOCTYPE service-group SYSTEM "avahi-service.dtd">
+
+<!-- See avahi.service(5) for more information about this configuration file -->
+
+<service-group>
+
+  <name replace-wildcards="yes">opentrons-%h</name>
```

Comment:
(Assuming this is the mDNS name of the service) The `opentrons-...` prefix no longer serves a purpose. Historically, it was added before we were advertising our own mDNS service and relying on the fact the Resin would also advertise it's SSH service (so we couldn't filter by looking for our own port).

If we don't want to use the `opentron-` prefix anymore, we don't have to

Reply:
I want to still because _http._tcp isn't just our service (although the port is unique, but still, it's not like we're advertising _opentrons.tcp like we perhaps ought to be).